contracts/factory/K1ValidatorFactory.sol

```diff
@@ -76,7 +97,7 @@ contract K1ValidatorFactory is Stakeable {
 
         // Create the validator configuration using the Bootstrap library
         BootstrapConfig memory validator = BootstrapLib.createSingleConfig(K1_VALIDATOR, abi.encodePacked(eoaOwner));
-        bytes memory initData = BOOTSTRAPPER.getInitNexusWithSingleValidatorCalldata(validator);
+        bytes memory initData = BOOTSTRAPPER.getInitNexusWithSingleValidatorCalldata(validator, REGISTRY, attesters, threshold);
```

Comment:
this would set the registry and also add the attesters during bootstrap?
and we can pass 0 addresses for registry, array of address 0 for attesters and threshold also 0.

Reply:
correct